lib/network.go

```diff
@@ -6371,6 +6371,16 @@ func DeserializePubKeyToUint64Map(data []byte) (map[PublicKey]uint64, error) {
 // MessagingGroupMetadata
 // ==================================================================
 
+// MessagingGroupOperation represents V3 Group Chat Messages ExtraData["MessagingGroupOperationType"] values
+type MessagingGroupOperation byte
+
+const (
+	MessagingGroupOperationAddMembers MessagingGroupOperation = iota
+	MessagingGroupOperationRemoveMembers
+	MessagingGroupOperationMuteMembers
+	MessagingGroupOperationUnmuteMembers
+)
```

Comment:
Typically we don't use iota's, because if we accidentally change the order of declaration later, previous transactions won't work. For example, if we moved RemoveMembers to the end, it would confuse the connect logic. Let's explicitly set values to numbers

Reply:
I generally prefer using iota over setting all values manually, but since these go into ExtraData I agree with you that it's better to define them explicitly